### PR TITLE
Documentation NFTS pallet

### PR DESCRIPTION
**Pull request by filippoweb3**
This PRs aims to include in the wiki information about what it is possible to do with the new [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts).

### CODE REVIEW

File: docs/learn/learn-system-chains.md [hunk 1]
```diff
@@ -50,18 +50,17 @@ network, including amounts that have been transferred to other parachains. It is
 where they can transact, to mint and burn, to manage the on-chain asset.
 
 {{ polkadot: Statemint :polkadot }}{{ kusama: Statemine :kusama }} also supports non-fungible assets
-(NFTs) via the Uniques pallet, however a new NFT pallet is coming in 2023 with more advanced
-features.
+(NFTs) via the [Uniques pallet](https://polkadot.js.org/docs/substrate/extrinsics#uniques) and the
+new [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts). For mre information about
+NFTs see the [dedicated wiki page](./learn-nft.md).
```
Comment (profradha) [inline docs/learn/learn-system-chains.md]:
```suggestion
(NFTs) via the [Uniques pallet](https://polkadot.js.org/docs/substrate/extrinsics#uniques) and the
new [nfts pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts). For more information about
NFTs see the [dedicated wiki page](./learn-nft-pallets.md).
```

**Review by DrW3RK — COMMENTED** [review]
Made a couple of changes. The learn-nft page serves as a high level overview of NFTs and a brief into about the parachains/projects with NFT capabilities. Created learn-nft-pallets page to take a deep dive into the technical details of the NFT pallets.

**Comment by filippoweb3** [comment]
@DrW3RK I created the "NFTs" category within "Basic" in the sidebar. Nfts are mentioned already in the "Asset" page within "Tokens and Assets". My point is that since we now have two pages related to NFTs (and I expect we will build more on them), NFTs deserve their category. It will also be easier to find them. Let me know what you think.

File: docs/learn/learn-nft.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 id: learn-nft
 title: NFTs
-sidebar_label: NFTs
+sidebar_label: Introduction to NFTs
 description: An explanation of the NFT landscape in the Polkadot ecosystem.
 keywords: [NFT, non-fungible token, NFT 2.0]
 slug: ../learn-nft
 ---
```
Comment (Gravel927) [inline docs/learn/learn-nft.md:8]: Drücken Sie lange auf ein Element in der Zwischenablage, um es anzupinnen. Nicht angepinnte Elemente werden nach einer Stunde gelöscht.

File: docs/learn/learn-assets.md [hunk 3]
```diff
@@ -133,10 +133,10 @@ stays up to date. For more info, see the "Moving Assets between Chains in XCM" s
 
 ## Non-Fungible Assets
 
-Unlike fungible assets, the particular instance of a non-fungible asset (NFT) has meaning separate
-from another instance of the same class.
+Unlike fungible assets, the particular instance of a [non-fungible asset (NFT)](./learn-nft.md) has
+separate meaning from another instance of the same class.
 {{ polkadot: Statemint :polkadot }}{{ kusama: Statemine :kusama }} represents NFTs in the
-[Uniques pallet](https://crates.parity.io/pallet_uniques/index.html).
+[Uniques and Nfts pallets](./learn-nft-pallets.md).
```
Comment (profradha) [inline docs/learn/learn-assets.md]:
```suggestion
[Uniques and NFTs pallets](./learn-nft-pallets.md).
```

File: docs/learn/learn-nft-pallets.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+sidebar_label: NFT Pallets
+description: An overview of the functionalities of NFT Pallets in Polkadot Ecosystem.
+keywords: [NFT, non-fungible token, NFT 2.0, nfts, NFT pallets]
+slug: ../learn-nft-pallets
+---
+
+NFT [pallets](http://localhost:3000/docs/learn-extrinsics#pallets-and-extrinsics) allow substrate
+developers to easily implement NFT-related actions within their dApp.
+
+## NFTS Pallet
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
## NFTs Pallet
```

File: docs/learn/learn-nft-pallets.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+id: learn-nft-pallets
+title: NFT Pallets
+sidebar_label: NFT Pallets
+description: An overview of the functionalities of NFT Pallets in Polkadot Ecosystem.
+keywords: [NFT, non-fungible token, NFT 2.0, nfts, NFT pallets]
+slug: ../learn-nft-pallets
+---
+
+NFT [pallets](http://localhost:3000/docs/learn-extrinsics#pallets-and-extrinsics) allow substrate
+developers to easily implement NFT-related actions within their dApp.
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
NFT [pallets](learn-extrinsics.md#pallets-and-extrinsics) allow
developers to easily implement NFT-related actions within their dApp.
```

File: docs/learn/learn-nft-pallets.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+## NFTS Pallet
+
+### Current Implementations
+
+Nfts is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
+on Westmint that provides a multitude of functions to interact with NFTs.
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
NFTs is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
on Westmint which provides a multitude of functions to interact with NFTs.
```

File: docs/learn/learn-nft-pallets.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+The pallet comes with a new way to configure it, as well as configure collections and items.
+Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow to disable
+functionalities that are not needed in the runtime.
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]: By "it", do you mean "NFTs"?

```suggestion
The pallet comes with a new way to configure NFTs, as well as configure collections and items.
Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
functionalities that are not needed in the runtime.
```

File: docs/learn/learn-nft-pallets.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+There are features that common to the nft world such as:
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]:
```suggestion
Some features are common to the NFT world, such as:
```

File: docs/learn/learn-nft-pallets.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+
+- Buying an item up for sale.
+- Creating (i.e. mint) and burn (i.e. destroy) a collection of items or a single item (burning must
+  be signed either by the admin of the collection or the owner). The creation of an item usually
+  involves setting some attributes specific to that item (an this is actually what drives the value
+  of the item). The creation of a collection also involves the specification of the maximum number
+  of items the collection has (an this also will drive the value of the collection's items in the
+  future).
```
Comment (profradha) [inline docs/learn/learn-nft-pallets.md]: Not all NFTs are meant for value creation. For instance, Soul bound NFTs, POAPs. 

```suggestion
- Buying an item up for sale.
- Creating (i.e., mint) and burning (i.e., destroy) a collection of items or a single item (burning must
  be signed either by the admin of the collection or the owner). Creating an item usually
    involves setting some attributes specific to that item. Creating a collection also requires the specification of the maximum number of items.
```

**Comment by filippoweb3** [comment]
@DrW3RK I added some minor edits to the "Asset" page. I think the next round of edits coming from the video we will watch can be added in another PR. Let me know what you think.

File: docs/learn/learn-assets.md [hunk 10]
```diff
@@ -57,26 +48,28 @@ required deposit of
 The network reserves the deposit on creation. The creator also must specify a unique `AssetId`, an
 integer of type `u32`, to identify the asset. The `AssetId` should be the canonical identifier for
 an asset, as the chain does not enforce the uniqueness of metadata like "name" and "symbol". The
-creator must also specify a minimum balance, which will prevent accounts from having dust balances.
+creator must also specify a minimum balance, preventing accounts from having dust balances.
 
-An asset class has a number of privileged roles. The creator of the asset automatically takes on all
+Asset classes and instances can have associated metadata. The metadata is an array of data that the
+class Owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting
+service. The [Uniques pallet](./learn-nft-pallets.md) also supports setting key/value pairs as
```
Comment (profradha) [inline docs/learn/learn-assets.md]:
```suggestion
service. The [Uniques pallet](./learn-nft-pallets.md#uniques-pallet) also supports setting key/value pairs as
```

File: docs/learn/learn-assets.md [hunk 11]
```diff
@@ -57,26 +48,28 @@ required deposit of
 The network reserves the deposit on creation. The creator also must specify a unique `AssetId`, an
 integer of type `u32`, to identify the asset. The `AssetId` should be the canonical identifier for
 an asset, as the chain does not enforce the uniqueness of metadata like "name" and "symbol". The
-creator must also specify a minimum balance, which will prevent accounts from having dust balances.
+creator must also specify a minimum balance, preventing accounts from having dust balances.
 
-An asset class has a number of privileged roles. The creator of the asset automatically takes on all
+Asset classes and instances can have associated metadata. The metadata is an array of data that the
+class Owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting
```
Comment (profradha) [inline docs/learn/learn-assets.md]:
```suggestion
class owner can add on-chain, for example, a link to an IPFS hash or other off-chain hosting
```

**Review by DrW3RK — APPROVED** [review]
Looks good!

File: docs/learn/learn-nft-pallets.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+- Delegating accounts: delegated accounts can approve changes to an item's attributes and transfer
+  an item. The item owner always has control and can decide to cancel approvals from a delegated
+  account.
+- One can also execute pending atomic swaps created by a counterpart.
+
+### Work in Progress
+
+[NFTs fractionalization](https://github.com/paritytech/substrate/pull/12565) will allow the user to:
+
+- Take ownership of an NFT from the [pallet-uniques](#uniques-pallet)
```
Comment (jsidorenko) [inline docs/learn/learn-nft-pallets.md:78]: Should be: [pallet-nfts](#nfts-pallet)

Comment (profradha) [reply docs/learn/learn-nft-pallets.md:78]: Updated! For the next PR, we will send it for your review before merging into the master.

**Pull request merged**